buildah.go

```diff
@@ -19,9 +19,17 @@ const (
 	// identify working containers.
 	Package = "buildah"
 	// Version for the Package
-	Version       = "0.1"
+	Version = "0.1"
+	// The value we use to identify what type of information, currently a
+	// serialized Builder structure, we are using as per-container state.
+	// This should only be changed when we make incompatible changes to
+	// that data structure, as it's used to distinguish containers which
+	// are "ours" from ones that aren't.
 	containerType = Package + " 0.0.1"
```

Comment:
We're talking about the parts of the structure that we pick up from other packages (the OCIv1 configuration structure), right?  That's defined as part of the image-spec, and I guess we could stand to call it out.